### ISSUE TITLE
Add dropout parameters to model

### ISSUE THREAD

**Issue by gb96** [gb96/GRUV#7]
Dropout between model layers is a regularization technique for reducing overfitting.

Allow two distinct dropout parameters dropout_W and dropout_U:
- dropout_W: float between 0 and 1. Fraction of the input units to drop for input gates.
- dropout_U: float between 0 and 1. Fraction of the input units to drop for recurrent connections.

See Keras [GRU](https://keras.io/layers/recurrent/#gru) and [LSTM](https://keras.io/layers/recurrent/#lstm) documentation.

Parameters to be specified in [config.nn_config](https://github.com/gb96/GRUV/blob/master/config/nn_config.py) and used when creating the model.
